<!DOCTYPE html>
<html><head><title>_requestFailedException</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">1 write to _requestFailedException</div><div class="rA">Azure.AI.TextAnalytics (1)</div><div class="rG" id="Azure.AI.TextAnalytics"><div class="rF"><div class="rN">AnalyzeOperation.cs (1)</div>
<a href="../AnalyzeOperation.cs.html#214"><b>214</b><i>_requestFailedException</i> = await ClientCommon.CreateExceptionForFailedOperationAsync(async, _diagnostics, _response, update.Value.Errors).ConfigureAwait(false);</a>
</div>
</div>
<div class="rH">2 references to _requestFailedException</div><div class="rA">Azure.AI.TextAnalytics (2)</div><div class="rG" id="Azure.AI.TextAnalytics"><div class="rF"><div class="rN">AnalyzeOperation.cs (2)</div>
<a href="../AnalyzeOperation.cs.html#52"><b>52</b>throw <i>_requestFailedException</i>;</a>
<a href="../AnalyzeOperation.cs.html#216"><b>216</b>throw <i>_requestFailedException</i>;</a>
</div>
</div>
</body></html>